wpengine-phpcompat.php

```diff
@@ -148,6 +149,21 @@ function check_status() {
 	}
 
 	/**
+	 * Remove all database options from the database.
+	 *
+	 * @since 1.3.2
+	 * @action wp_ajax_wpephpcompat_clean_up
+	 */
+	function clean_up() {
```

Comment:
Generally, I prefer to see this conditional expressed as a ! ( stuff ) { return } ; with the remaining logic outside of the conditional.  There is nothing wrong with the way it is now, but that is the WP norm for methods hooked on AJAX endpoints.  If this method was called outside of a CRON via the browser, wp_send_json() would never fire which contains the die(0);